dev/TabView/TabView.xaml

```diff
@@ -219,9 +222,84 @@
         </Setter>
     </Style>
 
+    <Style x:Key="TabViewCloseButtonStyle" TargetType="Button">
+        <Setter Property="BorderThickness" Value="0"/>
+        <Setter Property="Padding" Value="0"/>
+        <Setter Property="HorizontalAlignment" Value="Left"/>
+        <Setter Property="VerticalAlignment" Value="Center"/>
+        <Setter Property="FontFamily" Value="Segoe MDL2 Assets"/>
+        <Setter Property="FontWeight" Value="Normal"/>
+        <Setter Property="FontSize" Value="{ThemeResource ControlContentThemeFontSize}"/>
+        <Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}"/>
```

Comment:
@kaiguo is this the correct corner radius resource to use?

Reply:
I believe I had discussions with @chigy and tabs were considered as overlays, so that should be right resource to use.

Reply:
Just to confirm, overlay radius is correct for all 4 corners of the tab close button?

Reply:
Oh I see, looks like we didn't specify CornerRadius on close button [before](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/TabView/TabView.xaml#L535-L580), so the value should be 2 by default. We probably need to remove that line if we want to keep that value the same as before. Thanks for pointing it out!

Reply:
@YuliKl and @kaiguo , just double check. Tabs 4px, close button inside 2px, right?

Reply:
Correct.

Reply:
@kaiguo, what is the correct ThemeResource for @jtippet to use in this style to get a 2px corner radius?

Reply:
ControlCornerRadius